src/components/ArchivedReportFooter.js

```diff
@@ -90,5 +91,10 @@ export default compose(
         policies: {
             key: ONYXKEYS.COLLECTION.POLICY,
         },
+        reportClosedAction: {
+            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report.reportID}`,
+            canEvict: false,
+            selector: ReportActionsUtils.getLastClosedReportAction,
```

Comment:
This PR caused this regression https://github.com/Expensify/App/issues/16324 because the `getLastClosedReportAction` has been returning `undefined` which in turn did not get swapped for the defaultProps